packages/govuk-frontend/src/govuk/components/character-count/character-count.mjs

```diff
@@ -108,6 +121,12 @@ export class CharacterCount extends GOVUKFrontendComponent {
       datasetConfig
     )
 
+    if (!this.validateConfig(this.configSchema, this.config)) {
```

Comment:
This line is brill. Do we need to error on unknown properties too?

I'm assuming `{ maxwords: 20, colin: true }` will be valid

Reply:
Ideally, yes - but was holding off till we've figured out how to do our config schema/types.

Or do you think it's best to define the entire schema now within the component and move it later?

Reply:
I'm happy to leave it for another PR